cmd/config/cache/help.go

```diff
@@ -50,5 +50,11 @@ var (
 			Optional:    true,
 			Type:        "sentence",
 		},
+		config.HelpKV{
+			Key:         After,
+			Description: `minimum number of accesses before caching an object`,
```

Comment:
```suggestion
			Description: `minimum number of access before caching an object`,
```

Since we already indicate `number of` it is not contextually correct to call it `accesses`. If `accesses` is desired following wording makes sense

```suggestion
			Description: `minimum accesses before caching an object`,
```